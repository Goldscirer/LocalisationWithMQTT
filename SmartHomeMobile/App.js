import React, { Component } from 'react';
import { View, Text } from 'react-native';
import {Button, Header} from './src/Components/common';
import { ScrollView } from 'react-native';

class App extends Component {
  render() {
    return (
        <View>
          <Header headerText={'SMART HOME'} />
            <ScrollView>
                <Button/>
            </ScrollView>
        </View>
    );
  }
}

export default App;
