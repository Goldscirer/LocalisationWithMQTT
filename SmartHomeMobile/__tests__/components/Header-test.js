/**
 * @format
 */

import 'react-native';
import React from 'react';
import App, {Header} from '../../src/Components/common/Header';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

test('renders header', () => {
    const snap = renderer.create(
        <Header/>
    ).toJSON();
expect(snap).toMatchSnapshot();
});