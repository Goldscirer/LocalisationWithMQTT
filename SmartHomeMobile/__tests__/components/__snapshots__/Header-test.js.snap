// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#F8F8F8",
      "elevation": 5,
      "height": 40,
      "justifyContent": "center",
      "paddingTop": 5,
      "position": "relative",
      "shadowColor": "#000",
      "shadowOffset": Object {
        "height": 20,
        "width": 0,
      },
      "shadowOpacity": 0.2,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  />
</View>
`;
